### DISCUSSION TITLE
Starting Meson container error

### DISCUSSION THREAD

**Discussion by peercoder** [engageub/InternetIncome#134] [General]
Hi,
How fix this error?
Starting Meson container..
docker: Error response from daemon: conflicting options: port publishing and the container type network mode.

Thanks!

**Comment by engageub** [comment]
Hello,
Meson container does not work with proxies.
You have to use direct connection or Multi IPs.
If Port Forwarding is available on VPN it may work. 
Port has to be opened on the specific IP address to work. 
Do not use proxies with Meson. 

Thank you
